### PR TITLE
Fix tests for governancestaking

### PR DESCRIPTION
**Pull request by aliberkhsl**
#101

### CODE REVIEW

File: test/governance/GovernanceStaking.test.ts [hunk 1]
```diff
@@ -150,20 +150,24 @@ describe("GovernanceStaking - staking with GD  and get Rewards in GDAO", () => {
       [ethers.utils.parseEther("1728000")]
     );
     await ictrl.genericCall(governanceStaking.address, encodedCall, avatar, 0);
-
-    expect(await governanceStaking.rewardsPerBlock()).to.equal(
+    const rewardsPerBlock = await governanceStaking.rewardsPerBlock();
+    expect(rewardsPerBlock).to.equal(
       ethers.utils.parseEther("1728000").div(BN.from("518400"))
```
Comment (sirpy) [inline test/governance/GovernanceStaking.test.ts:155]: hardcoded and no explanation

File: test/governance/GovernanceStaking.test.ts [hunk 2]
```diff
@@ -342,13 +381,16 @@ describe("GovernanceStaking - staking with GD  and get Rewards in GDAO", () => {
...
+    const stakerProductivity = await governanceStaking.getProductivity(
+      staker.address
+    );
     expect(await governanceStaking.balanceOf(founder.address)).to.equal(0);
     expect(await governanceStaking.balanceOf(staker.address)).to.equal(100);
 
     expect((await governanceStaking.users(staker.address)).amount).to.equal(
       100
     );
+    expect(stakerProductivity[0]).to.be.equal("100");
```
Comment (sirpy) [inline test/governance/GovernanceStaking.test.ts:393]: you didnt check stakerproductivity before the transfer

**Review by sirpy — CHANGES_REQUESTED** [review]

File: test/governance/GovernanceStaking.test.ts (hunk 1 above)
Comment (aliberkhsl) [reply test/governance/GovernanceStaking.test.ts]: Added explanation

File: test/governance/GovernanceStaking.test.ts (hunk 2 above)
Comment (aliberkhsl) [reply test/governance/GovernanceStaking.test.ts:397]: Added this check as well

**Pull request merged**